ansible/roles/gemini/tasks/main.yml

```
@@ -0,0 +1,101 @@
+
+- name: "Build Gemini custom Docker image"
+  community.docker.docker_image:
+    name: gemini-sandbox
+    tag: latest
+    source: build
+    build:
+      path: "{{ config_dir_abs_path }}/gemini"
+      dockerfile: Dockerfile.gemini-sandbox
+      platform: linux/amd64
+    state: present
+  when: ansible_facts.services['docker'] is defined and ansible_facts.services['docker'].status == 'active'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)